.github/scripts/directory.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
```

Comment:
This file can be removed as no longer required.

Reply:
deleted